package.json

```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.11.1",
     "eslint-plugin-react-hooks": "^1.6.0",
+    "faker": "^4.1.0",
```

Comment:
Where is this used?

Reply:
https://github.com/acl-services/paprika/pull/416 it is use on the stories for the table